fmts/css.go

```diff
@@ -0,0 +1,17 @@
+package fmts
+
+func init() {
+	const lang = "css"
+
+	register(&Fmt{
+		Name: "stylelint",
+		Errorformat: []string{
+			`%+P%f`,
```

Comment:
I'm sorry for additional comment but I think `%-P%f` is better.
By using `-` flag, first `renderer/style.css|| renderer/style.cs` line will be removed.